src/models/StandaloneSimilarity.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+from cleaning_functions import pandas_vector_to_list
```

Comment:
we should have a common way/method to convert the pandas vector (which nobody is using (sorry i could't help it haha)) 

this was already in place, written by someone else and i tried to keep it uniform as not to introduce the Xth way to do the conversion (we already have three or something)

which is the best way? should agree on one and make everyone use it